hasher-matcher-actioner/hmalib/common/tests/test_mappings.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import unittest
+
+from threatexchange.content_type.photo import PhotoContent
+from threatexchange.content_type.text import TextContent
+from threatexchange.content_type.video import VideoContent
+from threatexchange.signal_type.pdq import PdqSignal
+from threatexchange.signal_type.raw_text import RawTextSignal
+from threatexchange.signal_type.md5 import VideoMD5Signal
+
+from hmalib.common.config import HMAConfig
+from hmalib.common.models.tests.ddb_test_common import HMAConfigTestBase
+from hmalib.common.mappings import (
+    HMASignalTypeMapping,
+)
+
+
+class SignalTypeMappingInitTestCase(HMAConfigTestBase, unittest.TestCase):
+    def test_bootstrapped_values_work(self):
```

Comment:
blocking: Can you add a test case where someone starts with the boostrapped configs, and then overrides PDQ with enabled (writing to DB), and then overrides PDQ to disabled (writing to DB) and make sure the state of the whole system is expected the whole way?

Reply:
No longer necessary because we are doing migrations (adding configs) a different way. Check out [this commit](https://github.com/facebook/ThreatExchange/pull/992/commits/d2e0f584a8fad5f9094b0db3423fdfb899884b30) for details.